Core/DISET/private/GatewayService.py

```diff
@@ -15,26 +15,35 @@
 
 """
 
-__RCSID__ = "$id:"
 
 import sys
 import cStringIO
+import os
+
+# TODO: Remove ThreadPool later 
+useThreadPoolExecutor = False
+if os.getenv('DIRAC_USE_NEWTHREADPOOL', 'NO').lower() in ('yes', 'true'):
+  from concurrent.futures import ThreadPoolExecutor
+  useThreadPoolExecutor = True
+else:
+  from DIRAC.Core.Utilities.ThreadPool import ThreadPool
+  from DIRAC.Core.DISET.private.MessageBroker import MessageBroker, getGlobalMessageBroker
```

Comment:
Is this import location in the `else` correct? It seems to me that the MessageBroker and getGlobalMessageBroker are use regardless of OldThreadPool or NewThreadPool given lines 96ff in this file. Or maybe lines 96ff need updating.

Reply:
Ops. You are right. I did a mistake when I cherry-pick (I did not resolved the conflicts correctly) from integration to v7r1:
https://github.com/zmathe/DIRAC/blob/threadPool/Core/DISET/private/GatewayService.py#L25

Reply:
I will run a test later today again to make sure everything is correct

Reply:
I was wondering if we can add the integrations tests with DIRAC_USE_NEWTHREADPOOL=yes to the actions

Reply:
I think you can do it using something like this: https://github.com/DIRACGrid/DIRAC/commit/1a3a5b3bfe843f57b0f53fe7590e7c10df598255

Reply:
I need to run with the flag the following test :Integration tests / Integration (5.7, slc6) (pull_request)

Reply:
I've done some local testing and an additional change is needed to propagated it to the `client` and `server` containers: d48342cff570509dd0ae2c1174f48e7f20202421

Reply:
Thanks. I have added your suggestion